Source/Core/Core/ConfigLoaders/NetPlayConfigLoader.cpp

```diff
@@ -105,6 +105,17 @@ class NetPlayConfigLayerLoader final : public Config::ConfigLayerLoader
 
       layer->Set(Config::MAIN_GCI_FOLDER_CURRENT_GAME_ONLY, true);
     }
+
+    // Check To Override Client's Gecko Codes
+    if (m_settings.m_SyncGeckoCodes)
+    {
+      if (!m_settings.m_IsHosting)
```

Comment:
This can be put into the parent `if` using `&&`.

Reply:
ok, this was a remnant from it being based off your save sync code. would you like me to adjust as that well?

https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/Core/ConfigLoaders/NetPlayConfigLoader.cpp#L94

Reply:
No, that has code in both the outer and inner if statements. This does not, making it redundant indentation.